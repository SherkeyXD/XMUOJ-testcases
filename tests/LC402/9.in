6
55763635059499467528
