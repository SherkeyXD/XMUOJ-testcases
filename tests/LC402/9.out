33059499467528
